docs/EVG.md

```diff
@@ -346,6 +346,12 @@ Each CA MUST publicly disclose its Certificate Policy and/or Certification Pract
 
 Effective as of 31 May 2018, the CA's Certificate Policy and/or Certification Practice Statement MUST be structured in accordance with RFC 3647. Prior to 31 May 2018, the CA's Certificate Policy and/or Certification Practice Statement MUST be structured in accordance with either RFC 2527 or RFC 3647. The Certificate Policy and/or Certification Practice Statement MUST include all material required by RFC 3647 or, if structured as such, RFC 2527.
 
+Effective as of 1 September 2020, the CA MUST publicly disclose through an appropriate and readily accessible online means that is available on a 24x7 basis every Incorporating Agency and Registration Agency it has determined satisfies the requirements in these Guidelines. The CA SHALL document where to obtain this information within Section 3.2 of the CA's Certificate Policy and/or Certification Practice Statement. This Agency Information SHALL include the following:
```

Comment:
September 1 is right around the corner for the disclosure of this list.  Would it be possible to split this into 2 phases?

1. Effective as of 1 September 2020, the CA MUST publicly disclose through an appropriate and readily accessible online means that is available on a 24x7 basis **AT LEAST 25% OF THE** Incorporating Agency and Registration Agency it has determined satisfies the requirements in these Guidelines.
2. Effective as of 1 February 2021, the CA MUST publicly disclose through an appropriate and readily accessible online means that is available on a 24x7 basis **EVERY** Incorporating Agency and Registration Agency it has determined satisfies the requirements in these Guidelines.

This way the CAs can set up the process and mechanics for disclosure "soon" and then have some more time to fully comply with this new requirement by listing every one and all of the accepted values associated with the source.

We might also want to have a clause that CAs can use a new one and then update the list within 30-60 days in case new sources are needed "urgently".  This would mean a back-dating of up to 30-60 days from use to the publishing of the new list would be permitted.

If the intent is to make this somewhat machine parseable, you might want to specify a common format so this can be processes similarly to how CCADB processes audit letters.

Reply:
Thanks Doug! This is great feedback, and it probably makes sense to discuss on our next call, but I’m really glad someone has looked.

Perhaps there’s some confusion to be addressed by wording, but the requirement is not to disclose what a CA has used retroactively, but to ensure disclosure going forward. September seemed eminently reasonable and there were no objections on the call, so is it possible this is just a misunderstanding?

We discussed the clause for exceptional circumstances, which was proposed by Wayne, and that seems to favor CAs over Relying Parties. I think we should continue discussion on our next validation call, because I do think it’s important to ensure any disclosures happen before issuance.

No, the intent is not to make it machine parsable, or to over engineer process. This was just trying to make some forward progress on getting the necessary disclosures so that we can move towards an industry-wide allowlist. This is simply a temporary step towards that, as CAs have otherwise not been forthcoming about what data sources they use, despite commitments on both CA Incidents and in the CABF.

This is also why it’s so important to have disclosure prior to issuance (so there is no uncertainty about whether a CA is still pending some disclosures) and a deadline within six months (e.g. September/Oct), so that we can begin compiling and discussing an allowlist within a year.

Reply:
Based on the call yesterday, there was general agreement among CAs that some flexibility on the completeness or timeliness of updates was needed for the September date.  Getting a disclosed list from all CAs is a big improvement over what we have today, even if the list is up to a month behind actual practices, imo.  Please consider updating the ballot to permit CAs up to 30 days to publish new entries in their "readily accessible online" system.

Reply:
As mentioned on the call, a month is an unacceptably unjustified delay and is not functionally better than nothing. We did talk about having total disclosure, sooner than September, while allowing a much smaller grace period (an upper bound of a week), and potentially allowing more time for automation to be implemented.